make-release.sh

```diff
@@ -107,19 +109,22 @@ set -e
 
 # change VERSION file
 echo "${VERSION}" > VERSION
-git add VERSION
 
 QUAY_REPO="quay.io/devfile/devworkspace-controller:${VERSION}"
 docker build -t "${QUAY_REPO}" -f ./build/Dockerfile .
 docker push "${QUAY_REPO}"
 
 set -x
 bash -x ./deploy/generate-deployment.sh --use-defaults --default-image quay.io/devfile/devworkspace-controller:${VERSION}
-# tag the release
-git tag "${VERSION}"
-git push origin "${VERSION}"
-COMMIT_MSG="[release] Release ${VERSION}"
-git commit -asm "${COMMIT_MSG}"
+
+# tag the release if the VERSION file has changed
+if [[ ! -z $(git status -s) ]]; then # dirty
+  COMMIT_MSG="[release] Release ${VERSION}"
+  git add VERSION
+  git commit -asm "${COMMIT_MSG}"
+  git tag "${VERSION}"
```

Comment:
I'm not sure but we may want to recreate tag.
But you know release process better

Reply:
tag recreation is controlled by GH action, before this script is called:

https://github.com/devfile/devworkspace-operator/blob/dontfailifnochanges/.github/workflows/release.yml#L22-L38